Octokit/Http/Connection.cs

```diff
@@ -485,6 +485,22 @@ Task<IApiResponse<T>> SendDataInternal<T>(object body, string accepts, string co
         }
 
         /// <summary>
+        /// Performs an asynchronous HTTP DELETE request that expects an empty response.
+        /// </summary>
+        /// <param name="uri">URI endpoint to send request to</param>
+        /// <param name="data">The object to serialize as the body of the request</param>
+        /// <param name="accepts">Specifies accept response media type</param>
+        /// <returns>The returned <seealso cref="HttpStatusCode"/></returns>
+        public async Task<HttpStatusCode> Delete(Uri uri,object data, string accepts)
```

Comment:
This will be handled by the next `FormatCode` call, so let's :boot: addressing this...